conversiontracerbullet/src/test/java/org/stanford/AuthDBPropertiesTest.java

```diff
@@ -28,23 +29,24 @@
 
     private AuthDBProperties authProps;
     private Properties serverConf;
-    private String propertyFile;
+    private String serverConfResourceName = "/server.conf";
     private File serverConfFile;
     private File tmpDir;
 
     @Before
     public void setUp() throws IOException, InvocationTargetException, IllegalAccessException, NoSuchMethodException, NoSuchFieldException {
+        tmpDir = createTempDirectory("AuthDBPropertiesTest_").toFile();
         authProps = new AuthDBProperties();
-        // Use private method to get the server.conf file path
-        Field f = AuthDBProperties.class.getDeclaredField("propertyFile");
-        f.setAccessible(true);
-        propertyFile = (String) f.get(authProps);
-        serverConfFile = new File(propertyFile);
-        // Use private method to get the server.conf properties
-        Method m = AuthDBProperties.class.getDeclaredMethod("loadDataSourceProperties", String.class);
+
+        Method m = AuthDBProperties.class.getDeclaredMethod("loadPropertyResource");
```

Comment:
If the tests could get direct access to the resource file without using the class...  Or if the tests could create the resource file dynamically...  but that's not possible, right?  Or it's more work.  Either way, there are tradeoffs and resource files have already given me a run around, in more ways than one (better explained in person than go on and on about it here).